integrations/tensorflow/iree_tf_compiler/TFL/test/lower_global_tensors.mlir

```diff
@@ -0,0 +1,25 @@
+// RUN: iree-opt-tflite -allow-unregistered-dialect -split-input-file -pass-pipeline='iree-tflite-lower-global-tensors' %s | IreeFileCheck %s
```

Comment:
Does this actually need `-allow-unregistered-dialect`?

```suggestion
// RUN: iree-opt-tflite -split-input-file -pass-pipeline='iree-tflite-lower-global-tensors' %s | IreeFileCheck %s
```

Reply:
Can you add more test cases here? In particular, there seem to be cases where you *don't* trigger a change, like if the resource ID of an assign op doesn't come from a handle op (I don't actually know what that means, but it's a codepath that I don't think is exercises). Additionally, it seems like you've got a few separate things that could be tested separately here. There are a few different "patterns" in the pass, that could have individual test cases (like just an assign)

Reply:
Should be more cases added now. It is difficult to factor things apart though as multiple cases must be true to declare a single global variable.